plugins/javaembed/HpccClassLoader.java

```diff
@@ -85,6 +85,13 @@ private HpccClassLoader(String classPath, ClassLoader parent, int _bytecodeLen,
         }
         return result; 
     }
+    public synchronized URL getResource(String path)
```

Comment:
if this is an override of the super method, let's annotate the method with a "@Override"
Also, I notice the method is synchronized, what are we protecting?

Reply:
I don't know enough java to know how to use @override...

Where does it go?

re the synchronized - I was going to cache them as the function above does, but decided not to. I can remove it.

Reply:
The annotations go above the method definition, in this case it would be:
@Override
public synchronized URL getResource(String path)